tests/cli/conftest.py

```diff
@@ -82,14 +83,25 @@ def mock_allocation_registry(testerchain, test_registry, mock_allocation_infile)
                                   client_password=INSECURE_DEVELOPMENT_PASSWORD,
                                   deployer_address=testerchain.etherbase_account)
 
+    if os.path.isfile(MOCK_ALLOCATION_REGISTRY_FILEPATH):
+        os.remove(MOCK_ALLOCATION_REGISTRY_FILEPATH)
+
     admin.deploy_beneficiaries_from_file(allocation_data_filepath=mock_allocation_infile,
                                          allocation_outfile=MOCK_ALLOCATION_REGISTRY_FILEPATH)
 
     allocation_registry = AllocationRegistry(filepath=MOCK_ALLOCATION_REGISTRY_FILEPATH)
+
     yield allocation_registry
+
+    # Cleanup Allocation Stuff
     if os.path.isfile(MOCK_ALLOCATION_REGISTRY_FILEPATH):
         os.remove(MOCK_ALLOCATION_REGISTRY_FILEPATH)
 
+    globs = ("allocation*.csv", "allocation*.json", "*ABI.json")
```

Comment:
Encapsulate into utility function, and use a temp dir for testing.

Reply:
a temp dir would be a pretty simple solution - then you can just delete the dir.

Reply:
:heavy_check_mark: